pennylane/devices/_qubit_device.py

```diff
@@ -1430,8 +1430,7 @@ def _samples_to_counts(self, samples, mp: CountsMP, num_wires):
         """Groups the samples into a dictionary showing number of occurences for
         each possible outcome.
 
-        The format of the dictionary depends on mp.return_type, which is set when
-        calling measurements.counts by setting the kwarg all_outcomes (bool). By default,
+        The format of the dictionary depends on whether the MeasurementProcess is Counts or AllCounts. By default,
```

Comment:
`AllCounts` is not a measurement process, it's a return type. The name of the MP is `CountsMP`.

Reply:
fixed at https://github.com/PennyLaneAI/pennylane/pull/6841/commits/d1df38fd076d27f34673b507af7a7a275221567f